movies/src/app.js

```diff
@@ -0,0 +1,32 @@
+const express = require('express');
+const entrypoint = require('./resources/entrypoint.json');
+const context = require('./resources/context.json');
+const doc = require('./resources/doc.json');
+
+const app = express();
+
+app.use(function(req, res, next) {
+  res.header('Access-Control-Allow-Origin', '*');
+  res.header('Access-Control-Expose-Headers', 'Location, Link');
+  res.header('Link', '</doc>; rel="http://www.w3.org/ns/hydra/core#apiDocumentation"');
+  next();
+});
+
+app.get('/', function (req, res) {
+  sendJsonLd(res, entrypoint);
+});
+
+app.get('/context.jsonld', function (req, res) {
+  sendJsonLd(res, context);
+});
+
+app.get('/doc', function (req, res) {
+  sendJsonLd(res, doc);
+});
+
+function sendJsonLd (res, resource) {
+  res.header("Content-Type", 'application/ld+json');
+  res.send(JSON.stringify(resource));
+}
```

Comment:
This is not necessary. You can use `express.static()` and any file with `jsonld` extension will be served with correct media type.

What's more, if you set `index: 'index.jsonld'` option then we can use URLs without extensions

Reply:
Will check that

Reply:
To get rid of the file endings I needed to add the option `extensions:['jsonld']`. But works fine now, thanks for the tip!